vllm/v1/attention/backends/flash_attn.py

```diff
@@ -469,7 +482,7 @@ def forward(
         num_actual_tokens = attn_metadata.num_actual_tokens
 
         # Handle encoder attention differently - no KV cache needed
-        if attn_type in (AttentionType.ENCODER_ONLY, ):
+        if attn_type in (AttentionType.ENCODER_ONLY, AttentionType.ENCODER):
```

Comment:
Do we need both `ENCODER_ONLY` and `ENCODER`

Reply:
It's a great question. I never found a case where they were different when it comes to AttentionType. It seems to be confusing the type of model with type of attention?

Combining them would be a good follow-up